### PR TITLE
Sroberge/split app

### PR DESCRIPTION
**Pull request by vim-sroberge**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,6 +58,9 @@
   "dependencies": {
     "dat.gui": "^0.7.7",
     "deepmerge": "^4.2.2",
+    "jsx": "^0.9.89",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```
Comment (mavimaec) [inline package.json:63]: @vim-sroberge was this intentionally placed in this repo? My understanding is that this repo (vim-webgl-viewer) is intended to define the API (without React) whereas the vim-webgl-app repo defines the demo app code (which can contain React).

Comment (vim-sroberge) [reply package.json:63]: Good catch I need to remove it.

File: tsconfig.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "jsx": "react",
```
Comment (mavimaec) [inline tsconfig.json:3]: Another little tidbit to remove regarding jsx/react

**Review by mavimaec — APPROVED** [review]
Thanks @vim-sroberge - approving to unblock (contingent on the removal of the React dependencies)

**Pull request merged**